server/src/controllers/ChapterUser/resolver.ts

```diff
@@ -264,6 +282,7 @@ export class ChapterUserResolver {
             user_chapters: { include: { chapter_role: true } },
           },
         },
+        chapter: true,
```

Comment:
```suggestion
        chapter: { select: { name: true } },
```
We only need the `name` after all.